### ISSUE TITLE
Confusing positive on prefer-over-quantifier

### ISSUE THREAD

**Issue by corbinu** [wix/eslint-plugin-lodash#152]
I am getting a positive on prefer-over-quantifier. May I am missing something but it seems like the over-quantifier version is more complicated than what it is detecting.

My code looks like this 
```
const ruralZone = _.find(zone.rurals, (rural) => {
return _.startsWith(rural, order.destination.postalCode) || _.startsWith(rural, order.destination.city);
});
```

I think the valid version would look something like this
```
const startWithRural = _.overSome([_.curryRight(_.startsWith, order.destination.postalCode), _.curryRight(_.startsWith, order.destination.city)])

const ruralZone = _.find(zone.rurals, startWithRural);
```

Which seems more complicated to me. If there is a better solution I am all for it, but perhaps the prefer-over-quantifier should only confirm if the constituent functions only take one input.

**Comment by ganimomer** [comment]
You're right. This really is more confusing.
Fixed in `v2.4.3`

**Comment by corbinu** [comment]
Thanks so much @ganimomer !
